core/internal/cltest/cltest.go

```diff
@@ -128,27 +129,6 @@ func init() {
 	gomega.SetDefaultEventuallyTimeout(3 * time.Second)
 	lvl := logLevelFromEnv()
 	logger.SetLogger(logger.CreateTestLogger(lvl))
-	// Register txdb as dialect wrapping postgres
```

Comment:
Can we remove this yet? Don't we need it for other tests still using the store?

Reply:
Actually I see it's in just moved to pgtest

Reply:
IT moved to pgtest